src/decimal.ts

```diff
@@ -0,0 +1,160 @@
+import { StateConverterOptionsWithContext } from "./converter";
+import { DecimalOptions } from "./converters";
+
+function normalizeLastElement(
```

Comment:
I think it would be nice if these functions specified their return value (string)